LockboxXCUITests/LockBoxScreenGraph.swift

```diff
@@ -0,0 +1,156 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import MappaMundi
+import XCTest
+
+class Screen {
+    static let WelcomeScreen = "WelcomeScreen"
+    static let LockboxMainPage = "LockboxMainPage"
+    static let SettingsMenu = "SettingsMenu"
+
+    static let FxASigninScreen = "FxASigninScreen"
+    static let FxCreateAccount = "FxCreateAccount"
+
+    static let OpenSitesInMenu = "OpenSitesInMenu"
+    static let LockedScreen = "LockedScreen"
+    static let AccountSettingsMenu = "AccountSettingsMenu"
+    static let AutolockSettingsMenu = "AutolockSettingsMenu"
+
+    static let SortEntriesMenu = "SortEntriesMenu"
+}
+
+class Action {
+    static let FxATypeEmail = "FxATypeEmail"
+    static let FxATypePassword = "FxATypePassword"
+    static let FxATapOnSignInButton = "FxATapOnSignInButton"
+    static let FxALogInSuccessfully = "FxALogInSuccessfully"
+
+    static let OpenSettingsMenu = "OpenSettingsMenu"
+
+    static let LockNow = "LockNow"
+
+    static let DisconnectFirefoxLockbox = "DisconnectFirefoxLockbox"
+    static let DisconnectFirefoxLockboxCancel = "DisconnectFirefoxLockboxCancel"
+
+    static let ChangeEntriesOrder = "ChangeEntriesOrder"
+    static let SelectAlphabeticalOrder = "SelectAlphabeticalOrder"
+    static let SelectRecentOrder = "SelectRecentOrder"
+}
+
+@objcMembers
+class LockboxUserState: MMUserState {
+    required init() {
+        super.init()
+        initialScreenState = Screen.WelcomeScreen
+    }
+
+    var fxaUsername: String? = nil
+    var fxaPassword: String? = nil
+}
+
+func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScreenGraph<LockboxUserState> {
+    let map = MMScreenGraph(for: test, with: LockboxUserState.self)
+
+    let navigationControllerBackAction = {
+        app.navigationBars.element(boundBy: 0).buttons.element(boundBy: 0).tap()
+    }
+
+    let fxaViewCancelButton = {
+        app.navigationBars["Lockbox.FxAView"].buttons["Cancel"].tap()
+    }
+
+    map.addScreenState(Screen.WelcomeScreen) { screenState in
+            screenState.tap(app.buttons["getStarted.button"], to: Screen.FxASigninScreen)
```

Comment:
I added two more, sorry that I didn't do at the same time than the first one, wasn't sure that the way I did it was correct. For the rest, I'm trying to get them but looks like I have to add them in a different way and have not found it so that they work yet.